xpublish_opendap/dap_xarray.py

```diff
@@ -16,10 +20,11 @@
     np.float32: dap.Float32,
     np.float64: dap.Float64,
     np.str_: dap.String,
-    # Not a direct mapping
-    np.int64: dap.Float64,
+    np.int64: dap.Float64,  # not a direct mapping
+}
+dtype_dap: Dict[np.dtype, dap.DAPAtom] = {
+    np.dtype(k): v for k, v in dtype_dap.items()
 }
```

Comment:
It looks like [mypy is failing](https://github.com/xpublish-community/xpublish-opendap/actions/runs/4833578323/jobs/8613781508?pr=3#step:4:76) because I redefined a variable that then you updated. If these can either be combined into a single definition or get different names, then I think that would make typing happy.

Reply:
I renamed the second variable, then deleted the first variable just to make sure the variable name-space is unambiguous (i.e., if someone had a debugger open).